### ISSUE TITLE
Autosuggest block cache tags

### ISSUE THREAD

**Issue by Ouzned** [Smile-SA/smile-magento-elasticsearch#44]
Hi,

I've noticed that the 3 blocks used by the autosuggest feature override the getCacheTags method: 

Smile_ElasticSearch_Block_Catalogsearch_Autocomplete_Suggest_Category
Smile_ElasticSearch_Block_Catalogsearch_Autocomplete_Suggest_Product
Smile_ElasticSearch_Block_Catalogsearch_Autocomplete_Suggest_Terms

```php
public function getCacheTags()
    {
        return array(Mage_CatalogSearch_Model_Query::CACHE_TAG);
    }
```
Because of this, the default cache group for blocks (block_html) is not added. As a result, it is impossible to delete this cache by just refreshing the html block cache, we have to delete the entire var/cache folder.

Could you modify these blocks to define a cache_tags attribute instead of overriding the method?

Thanks!

**Comment by romainruaud** [comment]
This will be added in future release.

Thank you for suggesting this.

Best regards,

**Comment by afoucret** [comment]
Fixed in release 1.3.
